.github/workflows/test.yml

```diff
@@ -4,21 +4,21 @@ on: [push, pull_request]
 
 jobs:
   test:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
```

Comment:
Can you pin it to a particular version of Ubuntu so that we can decide when to move?

Reply:
Yeah sure, the latest LTS I guess?

Reply:
Yeah that should be fine

Reply:
Done